atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1376,6 +1382,11 @@ private ConcurrentNavigableMap<Cell, byte[]> getLocalWrites(TableReference table
         return writesByTable.computeIfAbsent(tableRef, unused -> new ConcurrentSkipListMap<>());
     }
 
+    @VisibleForTesting
+    ConcurrentNavigableMap<Cell, ChangeMetadata> getChangeMetadataForWrites(TableReference tableRef) {
+        return metadataByTable.computeIfAbsent(tableRef, unused -> new ConcurrentSkipListMap<>());
+    }
+
```

Comment:
We could get away with not exposing this for tests, but it would not make our lives easier. (We'd have to assert on the lock request sent to timelock, which only contains lock descriptors. Ideally, we can use the table-and-cell abstraction in the tests)